test/Microsoft.AspNetCore.Mvc.IntegrationTests/MutableObjectModelBinderIntegrationTest.cs

```diff
@@ -1355,13 +1344,10 @@ private class Person9
             Assert.NotNull(model.Customer.Address);
             Assert.Equal(AddressStreetContent, model.Customer.Address.Street);
 
-            Assert.Equal(1, modelState.Count);
+            Assert.Equal(0, modelState.Count);
```

Comment:
I was referring to the assertion you _added_ @ [line 1350](https://github.com/aspnet/Mvc/pull/4010/files#diff-a17a31e64e5fd7e2f15d089bb238bed0R1350).